src/kusto/setup.py

```diff
@@ -9,9 +9,12 @@
 from codecs import open
 from setuptools import setup, find_packages
 
-# TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
 VERSION = '0.1.0'
+try:
+    from .manual.version import VERSION

```

Comment:
Should we bump version as well?

Reply:
Not sure what that means (and how to 'bump' the version)

Reply:
Increase the version number so that we can release new package.

Reply:
Increased to 0.1.1